src/support/err.c

```diff
@@ -503,25 +503,29 @@ __wt_panic_func(WT_SESSION_IMPL *session, int error, const char *func, int line,
     WT_CONNECTION_IMPL *conn;
     va_list ap;
 
-    conn = S2C(session);
+    /*
+     * !!!
+     * This function MUST handle a NULL WT_SESSION_IMPL handle.
+     *
+     * We will pedantically check the session and connection handles any time we use them.
+     */
+    if (session != NULL)
```

Comment:
This is more of a personal preference thing but I feel like it may be cleaner to use a ternary operator rather than an if else block. I'm fine with either way though.

Reply:
Changed in [f0b198a](https://github.com/wiredtiger/wiredtiger/pull/8154/commits/f0b198af47c2846efc4b2785a659525b5f368373)

I had originally considered the ternary operator and opted for the if ... else ... construction thinking it would be easier to follow for someone who doesn't know C well.  Maybe that shows my age. These days lots of other languages have the same ternary operator.